CHANGELOG.md

```diff
@@ -30,7 +36,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Fixed
 
-- Fixed a problem when there is a period in the profile name #43 #44
+- Fixed a problem when there is a period in the profile name #43, #44
```

Comment:
Yeah, I was just following the pattern. The numbers resolve as links when you look at the CHANGELOG tab for the extension in VS Code.

Reply:
You are right. Since it doesn't do the auto-linking in GitHub, I agree, we should explicitly include the links. I'll enter a new PR to fixup the others.